models/errors.go

```diff
@@ -2,6 +2,11 @@ package models
 
 import "fmt"
 
+const(
+	stBadParameterErrorMsg = "Bad value for parameter '%s': '%v'"
```

Comment:
It is used in the error test to ensure the message is correct. This is what @aslakknutsen complained about because it adds maintenance overhead. My hope is that it pays off in the long run.